packages/spacecat-shared-data-access/test/unit/models/configuration.test.js

```diff
@@ -55,6 +65,46 @@ describe('Configuration Model Tests', () => {
     expect(configuration.getJobs()).to.deep.equal(validData.jobs);
   });
 
+  it('checks if a handler type is enabled for a site', () => {
```

Comment:
This kind of unit tests should actually check that the value of `isEnabled` is as expected.
Also, I'd expect them to cover for all branches in the given method, e.g. enabledByDefault false, or site is included in sites, but org isn't in orgs etc.